### DISCUSSION TITLE
Applying a single rule without modifying the file indentation

### DISCUSSION THREAD

**Discussion by injarn** [rectorphp/rector#7887] [Q&A] [answered]
I tried using Rector by applying only the rule `$rectorConfig->rule(CurlyToSquareBracketArrayStringRector::class);`. However, too many files are being modified, most of them due to code indentation issues. Is there a way to apply the specified rule without modifying the indentation and avoid having too many files needlessly modified? Thank you.
I cannot use the `--dry-run` option because the project is very large and the number of files to be checked is high

**Comment by samsonasik** [comment, marked as answer]
That's known drawback, you can use coding standard tool for that https://github.com/rectorphp/rector#known-drawbacks
